libs/components/core/src/lib/modules/percent-pipe/percent.pipe.ts

```diff
@@ -11,54 +11,60 @@ import { SkyNumberFormatUtility } from '../shared/number-format/number-format-ut
   pure: false,
 })
 export class SkyPercentPipe implements OnDestroy, PipeTransform {
-  private defaultFormat = '1.0-2';
+  public get defaultLocale(): string {
+    return this.#defaultLocale;
+  }
+
+  #defaultFormat = '1.0-2';
 
-  private format: string;
+  #defaultLocale = 'en-US';
```

Comment:
This one is a bit different. The getter is more or less making the `#defaultLocale` a readonly value. We don't want to put a setter for this property since it should only be set by the locale provider.